astropy/io/fits/tests/test_division.py

```diff
@@ -27,7 +27,7 @@ def test_card_with_continue(self):
 
     def test_valid_hdu_size(self):
         t1 = fits.open(self.data('tb.fits'))
-        assert type(t1[1].size) == type(1)
+        assert isinstance(t1[1].size, type(1))
```

Comment:
I think here the test is for strict equality (though it should be `is` instead of `==`).